lib/pybytes.js

```diff
@@ -0,0 +1,169 @@
+'use babel';
+
+/* eslint-disable no-undef */
+const $ = require('jquery');
+const fs = require('fs');
+
+const rootElement =
+  '<div class="pybytes" style="height: 100%; width: 300px" id="pybytes-root"></div>';
+
+export default class Pybytes {
+  constructor(api) {
+    this.token = '';
+    this.isLoadingLogin = false;
+    this.view = $(rootElement).html();
+    this.panel = null;
+    this.api = api;
+    this.staySignedIn = false;
+    this.loginButton = $('#pybytes-login-button');
+    this.loginEmailInput = $('#pybytes-login-email');
+    this.loginPasswordInput = $('#pybytes-login-password');
+    this.loginPath = `${this.api.getPackageSrcPath()}/views/pybytes-panel.html`;
+    this.homePath = `${this.api.getPackageSrcPath()}/views/pybytes-panel-home.html`;
+  }
+
+  render() {
+    let HTML = null;
+
+    const plainHtml = fs.readFileSync(
+      this.token ? this.homePath : this.loginPath,
+    );
+    if ($('#pybytes-root').length) {
+      HTML = $('#pybytes-root').html(plainHtml.toString());
+    } else HTML = $(rootElement).html(plainHtml.toString());
+    this.view = HTML;
+    this.bindingEventsHome();
+    return HTML;
+  }
+
+  bindingEventsLogin() {
+    this.loginButton = $('#pybytes-login-button');
+    this.loginEmailInput = $('#pybytes-login-email');
+
+    this.loginEmailInput.keydown(event => {
+      if (event.key === 'Tab') {
+        this.loginPasswordInput.focus();
+      } else if (event.key === 'Enter') {
+        this.login();
+      }
+    });
+
+    this.loginPasswordInput = $('#pybytes-login-password');
+
+    this.loginPasswordInput.keydown(event => {
+      if (event.key === 'Tab') {
+        this.loginEmailInput.focus();
+      } else if (event.key === 'Enter') {
+        this.login();
```

Comment:
I understand this is just a demo Pybytes code, but I saw that `login` function is async.

Then in the places where this function is used, you're not using `await` keyword.